src/modules/70-pipeline/components/PipelineStudio/TemplateBar/TemplateBar.tsx

```diff
@@ -213,15 +214,31 @@ export function TemplateBar(props: TemplateBarProps): JSX.Element {
       <Layout.Horizontal spacing={'small'} flex={{ alignItems: 'center', justifyContent: 'flex-start' }}>
         <Icon size={11} color={Color.WHITE} name={'template-library'} />
         {loading && (
-          <Text style={{ flexGrow: 1 }} font={{ size: 'small' }} color={Color.WHITE}>
+          <Text font={{ size: 'small' }} color={Color.WHITE}>
             {getString('loading')}
           </Text>
         )}
         {!loading && !isEmpty(selectedTemplate) && (
-          <Text style={{ flexGrow: 1 }} font={{ size: 'small' }} color={Color.WHITE}>
+          <Text
+            font={{ size: 'small' }}
+            color={Color.WHITE}
+            lineClamp={1}
+            className={selectedTemplate?.storeType === 'INLINE' ? css.inlineText : ''}
+          >
             {`Using Template: ${getTemplateNameWithLabel(selectedTemplate)}`}
           </Text>
         )}
+        {selectedTemplate?.storeType === 'REMOTE' && (
```

Comment:
import { StoreType } from '@common/constants/GitSyncTypes'
```suggestion
        {selectedTemplate?.storeType === StoreType.REMOTE && (
```